README.md

```diff
@@ -65,19 +140,42 @@ class PurchaseScreen < PM::Screen
 end
 ```
 
-#### purchase_iap(*product_ids, &callback)
+#### retrieve_iaps(*product_ids, &callback)
+
+Retrieves in-app purchase products in an array of mapped hashes. The callback method should accept `products` and `error`.
```

Comment:
The `*` screws this up. Might need to add backticks around the methods to avoid issues.